Logos/ForgeRock_Vert_Color_Logo_RGB_R_med.svg

```diff
@@ -1,5 +1,4 @@
-<svg id="ede8ca5d-677a-4786-83dc-809e9b4a1e06" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="2.68003in" height="1.44797in" viewBox="0 0 192.96185 104.25351">
-
+<svg id="ede8ca5d-677a-4786-83dc-809e9b4a1e06" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="800px" height="800px" viewBox="0 0 192.96185 104.25351">
   <title>ForgeRock_Vert_Color_Logo_RGB_R_med</title>
```

Comment:
Invalid Logo, Please follow the guidelines provided here: https://github.com/Azure/Azure-Sentinel/tree/master/DataConnectors

e company logo adhering to the following requirements in the 'Logo' folder
Logo needs to be in SVG format and under 5 Kb
Ensure raw file of logo does not have any of the following:
cls and style formats
embedded png formats
xmlns:xlink
data-name
Do not use xlink:href - use inline instead
Do not use title tag
If some properties in the logo have IDs (for e.g. <g id="layer0"...), then set these IDs as GUIDs so that these are uniquely identifiable across all Azure logo assets
Logo scales well to fit in a 75 px square
SVG code file is formatted well for readability